src/us/kbase/common/service/ServiceChecker.java

```diff
@@ -21,6 +21,7 @@
 	public static final int PY_CODE = -32700;
 	public static final String JV_PL_MSG = "HTTP GET not allowed.";
 	public static final int JV_PL_CODE = -32300;
+	public static final String HandleSerivce_MSG = "No JSON object could be decoded";
```

Comment:
Was the handle service built with an older version of the SDK? I just had to change the JSON object line to the PY_MSG line above when the catalog was updated to python3: 
https://github.com/kbase/workspace_deluxe/commit/c04d5a809dcea0e942a6dab52f1de598759cf1ac

Reply:
hmmm handle service was build with latest SDK in python3

Reply:
```
curl https://ci.kbase.us/services/catalog
{"error": {"code": -32700, "name": "Parse error", "message": "Expecting value: line 1 column 1 (char 0)", "error": null}, "version": "1.1"}
```

Reply:
Maybe check with James on this? I don't see why the catalog and HS should give different errors for the GET verb.

Reply:
👍 I got it fixed running Python3 locally and on travis.